.changelog/E05059DF-7A3D-4F8E-888A-7883D0ABABDB.json

```diff
@@ -0,0 +1,9 @@
+{
+    "id": "E05059DF-7A3D-4F8E-888A-7883D0ABABDB",
+   "type": "bugfix",
+    "description": "The S3 Uploader is updated to enabled to request checksums for multipart uploads only if RequestChecksumCalculation is not set to `when_required`. This makes it possible to preserve backwards compatibility for multipart uploads for third-party S3 providers.",
+    "collapse": false,
+    "modules": [
+        "service/s3",
```

Comment:
wait, are you sure this module is "service/s3" instead of "feature/s3/manager"

Reply:
Is there some script that does this? The only information about generating these entries I could find:

* https://github.com/aws/aws-sdk-go-v2/blob/0475b822f1f48ca746369818d51272f989c06b91/Makefile#L487-L492

I see the instructions are now removed in https://github.com/aws/aws-sdk-go-v2/commit/5b11c8d01f258aa7641c77c5dd0f97a2ad3d73c9. So I'm rather confused about whether these entries are needed by outside contributors.

Reply:
@stanhu yes, could you regenerate a new changelog following our [doc](https://github.com/awslabs/aws-go-multi-module-repository-tools/blob/main/cmd/changelog/README.md)

Reply:
@wty-Bryant Done.